app/src/styles/base.css

```
@@ -398,6 +398,17 @@ h1, h2, h3, h4 { font-family: var(--font-heading); white-space: nowrap; margin:
   .skeleton { animation: none; }
 }
 
+/* Boot splash mark: a slow breathing scale so the screen reads as "working"
+ * rather than frozen during the (usually sub-second) session restore. */
+@keyframes markBreathe {
```

Stylelint (17.14.0)

[error] Expected keyframe name "markBreathe" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)